src/couch_db_updater.erl

```diff
@@ -605,10 +605,41 @@ init_db(DbName, Filepath, Fd, Header0, Options) ->
             Db
     end.
 
+open_db_file(Filepath, Options) ->
```

Comment:
@iilyak It seems there are some commits that are not related to multipart refactor. Could you rebase your PR against current master?